extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -330,6 +327,12 @@ export class AmpA4A extends AMP.BaseElement {
      * {?HTMLIframeElement}
      */
     this.iframe = null;
+
+    /**
+     * @return whether most recent ad request was generated as part of resume
```

Comment:
You mean `@type {???}`. Can you also annotate it with `TODO` for future cleanup?